src/containers/HomePage.js

```diff
@@ -1,18 +1,43 @@
 import React, { Component } from 'react';
 // import { connect } from 'react-redux';
 import { Link } from 'react-router';
+import Header from './Header';
 
 export default class Home extends Component {
 
   render() {
     return (
-      <div>
-        <section id="HomePage">
-          <h1>Home</h1>
-          <Link to="/Settings">Pinned Cities</Link>
-          <Link to="/ExtendedForecast">Extended Forecast</Link>
+      <section id="HomePage">
+        <section className="CityContainer">
+          <article className="CityForecast">
+            <ul><h2 className="city-name">Current weather for Chicago</h2>
+              <li>Temperature: 64&#176; F</li>
+              <li>Weather: Partly Cloudy</li>
+              <li>Wind: 35mph</li>
+              <li>Humidity: 78%</li>
+            </ul>
+            <h3 className="ViewExtended">View Extended Forecast >></h3>
+          </article>
+          <article className="CityForecast">
+            <ul><h2 className="city-name">Current weather for Maui</h2>
```

Comment:
The `h2` inside of the `ul` feels super yucky. Consider moving it out of the list.